src/main/java/com/ldtteam/structurize/config/BlueprintRenderSettings.java

```diff
@@ -34,10 +54,11 @@ private BlueprintRenderSettings()
      */
     public BlueprintRenderSettings(final FriendlyByteBuf byteBuf)
     {
-        final int size = byteBuf.readInt();
-        for (int i = 0; i < size; i++)
+        renderSettings = new ArrayList<>(SharedSettings.VALUES.size());
+
+        for (final var setting : SharedSettings.VALUES)
```

Comment:
I really hate var, lets try to use the instance if we actually know it =D

Reply:
Yep, I'm like foreach are fine, but will switch